### PR TITLE
Updated board membership list

### PR DESCRIPTION
**Pull request by ehsteve**
Includes a number of other fixes such as
* moving to using the svg version of the logo.
* removing extra white spaces.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -134,10 +134,10 @@ <h1 id="acknowledging">Acknowledging or Citing SunPy</h1>
     <a href="https://www.zotero.org/groups/sunpy_-_python_for_solar_physicists">public list</a>
      of papers on <a href="https://www.zotero.org/">Zotero</a>.
 </li>
-   <li><b>For a poster, talks, or project websites</b>, please include the Sunpy logo 
+   <li><b>For a poster, talks, or project websites</b>, please include the <a href="https://raw.githubusercontent.com/ehsteve/sunpy-logo/master/generated/sunpy_logo.svg"></a>Sunpy logo</a> 
    on the title, conclusion slide, or about page. For websites please link the image to sunpy.org.<br>
-   <img src="https://github.com/sunpy/sunpy-logo/raw/master/generated/sunpy_powered_300x153.png"><br>
-   Other versions of the logo are also <a href="https://github.com/sunpy/sunpy-logo/">available</a>.</li>
+   <img width="300px" src="https://raw.githubusercontent.com/ehsteve/sunpy-logo/master/generated/sunpy_powered.svg"><br>
```
Comment (Cadair) [inline about.html]: these should not be pointing at your branch when this is merged.

**Comment by Cadair** [comment]
The links to your fork of the logo repo should be switched back to the sunpy one before this is merged.

**Comment by ehsteve** [comment]
@Cadair ping

**Comment by Cadair** [comment]
Sure.

**Pull request merged**